great_expectations/types/color_palettes.py

```diff
@@ -0,0 +1,47 @@
+from enum import Enum
```

Comment:
@NathanFarmer How come we put these `color palettes` into the global types directory (not saying that this is wrong, just curious as to the reasoning -- as opposed to something in the `rule_based_profiler` directory).  Thanks!